app/views/dashboards/index.html.erb

```diff
@@ -0,0 +1,65 @@
+
+<div class="text-center container-fluid">
+
+  <!-- My Games modal -->
+  <button type="button" class="btn btn-primary btn-lg" data-toggle="modal" data-target="#myModal1">
+    My Games
+  </button>
+
+  <!-- My Games Modal -->
+  <div class="modal fade" id="myModal1" tabindex="-1" role="dialog" aria-labelledby="myModalLabel">
+    <div class="modal-dialog" role="document">
+      <div class="modal-content">
+        <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+          <h4 class="modal-title" id="myModalLabel">My Games</h4>
+        </div>
+        <div class="modal-body">
+
+        </div>
+        <div class="modal-footer">
+          <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <!-- My Games modal -->
```

Comment:
Change this comment and the one below to `<!-- Open Games Modal -->` to avoid confusion.